spec/dummy/db/migrate/20170815023204_create_async_request_jobs.rb

```diff
@@ -0,0 +1,16 @@
+class CreateAsyncRequestJobs < ActiveRecord::Migration
+  def change
+    create_table :async_request_jobs do |t|
+      t.string :worker
+      t.integer :status
+      t.integer :status_code
+      t.text :response
```

Comment:
can we limit the amount of data here ? a commun text field can store 4GB of data

Reply:
Can't we use a Json field since we are storing it as `response.to_json`?

Reply:
maybe. But what do we do with the already existing responses. We should migrate those. Also you are assuming that every db supports json field. Do every db support that kind of field?

Reply:
Mmmm postgres, mysql, mariaDB supports it (except its a really old version), I don't know about Mongo, but this is not for NoSQL databases right ?

Reply:
But it also depends on the DB version, jsonb was added in postgres 9.x for example

Reply:
Then just limiting the size

Reply:
We are going to discuss this for another version, we can do a migration for this